services/sky/document_view.h

```diff
@@ -41,14 +35,13 @@ class TextureLayer;
 
 class DocumentView : public blink::ServiceProvider,
                      public blink::SkyViewClient,
-                     public mojo::ViewManagerDelegate,
-                     public mojo::ViewObserver,
+                     public mojo::NativeViewportEventDispatcher,
                      public sky::LayerClient,
                      public sky::LayerHostClient,
                      public ui::GestureConsumer {
  public:
-  DocumentView(mojo::InterfaceRequest<mojo::ServiceProvider> services,
-               mojo::ServiceProviderPtr exported_services,
+  DocumentView(mojo::InterfaceRequest<mojo::ServiceProvider> exported_services,
+               mojo::ServiceProviderPtr imported_services,
```

Comment:
These don't come from the embedder, they're coming from the shell so they're still relevant.  I just fixed the naming since the old names were wrong, particularly "exported_services" which should have been "exposed_services" as in Application.mojom.